specification/storagecache/resource-manager/Microsoft.StorageCache/preview/2020-08-13-aml-preview/examples/AscOperations_Get.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "613192d7-503f-477a-9cfe-4efc3ee2bd60",
+    "location": "West US",
+    "operationId": "testoperationid",
+    "api-version": "2020-03-01"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/id/locations/westus/ascOperations/testoperationid",
```

Comment:
Will we see arbitrary strings for operation names or UUIDs?

Reply:
Updated to match all the other examples.

Reply:
Sorry - I guess I wasn't clear. Are operation IDs really arbitrary strings, or are they UUIDs?

Reply:
Why do you care?   I'm leery of over-specifying this field.

Reply:
Reasonable. Resolving.